hypervisor/src/cpu.rs

```diff
@@ -1,12 +1,12 @@
 // Copyright © 2019 Intel Corporation
 //
-// SPDX-License-Identifier: Apache-2.0
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
```

Comment:
If you're dual licensing then this should be:

```
SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
```

You'd be using `AND` for describing that parts of the code are Apache-2.0 licensed and other parts are BSD licensed, which would makes the overall license both Apache-2.0 *AND* BSD licensed.

Here you want to give the future crate users a choice of license between Apache-2.0 *OR* BSD when reusing the code.

Reply:
@sameo Updated!